assets/css/edd-admin.css

```diff
@@ -62,19 +62,21 @@ a.edd-delete:hover {
 	outline: auto 5px -webkit-focus-ring-color;
 	border-color: #eee;
 	box-shadow: none;
+	position: relative;
 }
 
 .edd-from-to-wrapper input[name="start"],
 .edd-from-to-wrapper input[name="start-date"],
 .edd-from-to-wrapper input[name="filter_from"] {
 	border-right-color: #eee;
+	border-radius: 4px 0 0 4px;
 }
 
 .edd-from-to-wrapper input[name="end"],
 .edd-from-to-wrapper input[name="end-date"],
 .edd-from-to-wrapper input[name="filter_to"] {
-	border-left-color: #eee;
 	margin-left: -1px;
+	border-radius: 0 4px 4px 0;
```

Comment:
`:focus` styles need to be added for these as well--when the inputs have focus, the border-radius disappears. Actually, I think it's not that the border-radius is changed or removed, but the outline added to `.edd-from-to-wrapper input:focus` (line 60, I think), overrides it.

Reply:
I think I've got the focus fixed up, but what color should it be?  The `select`s focus in black while the search field is in the admin color scheme.  Again, I'm guessing it is a Chosen side effect.  What did you opt for in #7956?

Reply:
For me, it's picking up the admin color scheme focus now and I think everything looks good:

![image](https://user-images.githubusercontent.com/4853066/90652281-e43de280-e20b-11ea-9a94-821acf6f7e4a.png)